### DISCUSSION TITLE
Github page creation

### DISCUSSION THREAD

**Discussion by vivek280393** [community/community#28907] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,
    I am trying to create github site but i am not able to create it. Not sure what is going wrong so if someone can help me to fix the issue it will be appreciated. 

After saving the branch in main i am not getting the site built message so not sure why it is not getting created.

Thanks in advance 

Screenshot:
<img width="924" alt="github pages" src="https://user-images.githubusercontent.com/110919925/183692095-b323516e-4afc-4749-90b9-2e383a3a0bb0.PNG">

**Comment by airtower-luna** [comment, marked as answer]
Check the build log in the Actions tab. :wink: https://github.com/vivek280393/vivek280393.github.io/runs/7749418629?check_suite_focus=true#step:4:74

Specifically, the `build` jobs is failing because you've configured it to deploy from the `docs/` directory in your `main` branch, but there is no such directory. Looking at your repo I assume you want to actually deploy from the root instead.

**Reply by vivek280393** [reply]
Thank you i got it resolved now. I appreciate your timely help

@airtower-luna

**Reply by airtower-luna** [reply]
I'm glad to hear that! Marking the answer would be appreciated. :smile_cat:
